test/test-e2e.sh

```diff
@@ -78,6 +97,10 @@ run_tests() {
   go test test/e2e_test.go -v -count=1
 }
 
+upload_metrics_server_logs(){
+  ${KUBECTL} logs -n kube-system -l "k8s-app=metrics-server" > $ARTIFACTS/metrics-server.log
```

Comment:
It is a good idea to output the logs of metrics-server
However, there are fewer logs output by executing the `kubectl logs -n kube-system -l "k8s-app=metrics-server"` command.
We can add it first to analyze whether the log is enough

Reply:
are you talking about grabbing the audit logs as well?

Reply:
Don't think K8s guarantees that you will get all logs, but also I don't think there is a better way.

Reply:
I mean not sure if the logs obtained by kubectl logs -n kube-system -l "k8s-app=metrics-server" can meet the requirement.
I also want to analyze the metrics-server logs when CI fails, I think it can be helpful sometimes